core/events.go

```diff
@@ -86,6 +86,17 @@ type ChainInsertEvent struct {
 	LatestBlockTime time.Time
 }
 
+type ReceiptChainInsertEvent struct {
```

Comment:
This
- is used in the `Insert` return value type for `InsertReceiptChain` and useful for listening for and logging downloader fast sync progress.